app/src/main/res/values/style.xml

```diff
@@ -14,4 +14,26 @@
     <style name="links.gone" parent="links">
         <item name="android:visibility">gone</item>
     </style>
+
+    <style name="linksLarge">
+        <item name="android:layout_width">wrap_content</item>
```

Comment:
we could abstract out common properties into a parent style and inherit from it instead